diktat-rules/src/test/kotlin/org/cqfn/diktat/ruleset/chapter3/spaces/WhiteSpaceRuleWarnTest.kt

```diff
@@ -16,10 +16,11 @@ class WhiteSpaceRuleWarnTest {
 
     private val lbraceWarn = "${WRONG_WHITESPACE.warnText()} there should be a whitespace before '{'"
     private val eolSpaceWarn = "${WRONG_WHITESPACE.warnText()} there should be no spaces in the end of line"
-    private fun binaryOpWarn(op: String, before: Int?, after: Int?, reqBefore: Int, reqAfter: Int?) =
-//            "${WRONG_WHITESPACE.warnText()} $op should ${if (isException) "not " else ""}be surrounded by whitespaces"
-            "${WRONG_WHITESPACE.warnText()} $op should have $reqBefore space(s) before" +
-                    (if (reqAfter != null) " and $reqAfter space(s) after" else "") +
+    private fun tokenWarn(token: String, before: Int?, after: Int?, reqBefore: Int?, reqAfter: Int?) =
+            "${WRONG_WHITESPACE.warnText()} $token should have" +
```

Comment:
you know how I don't like such code with "if"s, but lgtm anyway

Reply:
Well, this method in for tests only, and there would be a lot of branches, so I thought that poor readability won't hurt much in this particular case :)